profiles/kanishbodhwani.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/64742906/192156828-e3660f3b-ee38-428b-bd93-f875169c6987.png",
+    "issueId": 1389,
+    "name": "Kanish Bodhwani",
+    "username": "kanishbodhwani"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```